lib/dry/view/exposure.rb

```diff
@@ -48,12 +48,17 @@ def private?
         options.fetch(:private) { false }
       end
 
+      def default_value
+        options[:default]
+      end
+
       def call(input, locals = {})
-        if proc
-          call_proc(input, locals)
-        else
-          input[name]
-        end
+        value = if proc
+                  call_proc(input, locals)
+                else
+                  input[name]
+                end
+        value || default_value
```

Comment:
@GustavoCaso what if `value` is actually meant to be `nil`? In this case it'd get overridden by the `default_value`. Reckon you can try finding a way to prevent this from happening?